plugins/meson.go

```diff
@@ -11,6 +11,7 @@ import (
 type MesonModule struct {
 	Name   string
 	Type   string
+	BuildFlags string            `json:"buildflags"`
```

Comment:
this should be a string array, not a single string

Reply:
thats not how the other modules nor the documentation are written.  want me to proceed with the array?

Reply:
yes, the documentation can still be changed